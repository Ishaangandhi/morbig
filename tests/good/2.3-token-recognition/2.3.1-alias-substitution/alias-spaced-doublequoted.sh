alias x="true "
x
x x
x x x
