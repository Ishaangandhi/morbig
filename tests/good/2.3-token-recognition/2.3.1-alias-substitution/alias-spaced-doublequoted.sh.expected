[
  "Program_LineBreak_CompleteCommands_LineBreak",
  [ "LineBreak_Empty" ],
  [
    "CompleteCommands_CompleteCommands_NewlineList_CompleteCommand",
    [
      "CompleteCommands_CompleteCommands_NewlineList_CompleteCommand",
      [
        "CompleteCommands_CompleteCommands_NewlineList_CompleteCommand",
        [
          "CompleteCommands_CompleteCommand",
          [
            "CompleteCommand_CList",
            [
              "CList_AndOr",
              [
                "AndOr_Pipeline",
                [
                  "Pipeline_PipeSequence",
                  [
                    "PipeSequence_Command",
                    [
                      "Command_SimpleCommand",
                      [
                        "SimpleCommand_CmdName_CmdSuffix",
                        [
                          "CmdName_Word",
                          [ "Word", "alias", [ [ "WordName", "alias" ] ] ]
                        ],
                        [
                          "CmdSuffix_Word",
                          [
                            "Word",
                            "x=\"true \"",
                            [
                              [
                                "WordAssignmentWord",
                                [
                                  [ "Name", "x" ],
                                  [
                                    "Word",
                                    "\"true \"",
                                    [
                                      [
                                        "WordDoubleQuoted",
                                        [
                                          "Word",
                                          "true ",
                                          [ [ "WordLiteral", "true " ] ]
                                        ]
                                      ]
                                    ]
                                  ]
                                ]
                              ]
                            ]
                          ]
                        ]
                      ]
                    ]
                  ]
                ]
              ]
            ]
          ]
        ],
        [ "NewLineList_NewLine" ],
        [
          "CompleteCommand_CList",
          [
            "CList_AndOr",
            [
              "AndOr_Pipeline",
              [
                "Pipeline_PipeSequence",
                [
                  "PipeSequence_Command",
                  [
                    "Command_SimpleCommand",
                    [
                      "SimpleCommand_CmdName",
                      [
                        "CmdName_Word",
                        [ "Word", "\"true \"", [ [ "WordLiteral", "x" ] ] ]
                      ]
                    ]
                  ]
                ]
              ]
            ]
          ]
        ]
      ],
      [ "NewLineList_NewLine" ],
      [
        "CompleteCommand_CList",
        [
          "CList_AndOr",
          [
            "AndOr_Pipeline",
            [
              "Pipeline_PipeSequence",
              [
                "PipeSequence_Command",
                [
                  "Command_SimpleCommand",
                  [
                    "SimpleCommand_CmdName_CmdSuffix",
                    [
                      "CmdName_Word",
                      [ "Word", "\"true \"", [ [ "WordLiteral", "x" ] ] ]
                    ],
                    [
                      "CmdSuffix_Word",
                      [ "Word", "\"true \"", [ [ "WordLiteral", "x" ] ] ]
                    ]
                  ]
                ]
              ]
            ]
          ]
        ]
      ]
    ],
    [ "NewLineList_NewLine" ],
    [
      "CompleteCommand_CList",
      [
        "CList_AndOr",
        [
          "AndOr_Pipeline",
          [
            "Pipeline_PipeSequence",
            [
              "PipeSequence_Command",
              [
                "Command_SimpleCommand",
                [
                  "SimpleCommand_CmdName_CmdSuffix",
                  [
                    "CmdName_Word",
                    [ "Word", "\"true \"", [ [ "WordLiteral", "x" ] ] ]
                  ],
                  [
                    "CmdSuffix_CmdSuffix_Word",
                    [
                      "CmdSuffix_Word",
                      [ "Word", "\"true \"", [ [ "WordLiteral", "x" ] ] ]
                    ],
                    [ "Word", "\" true \"", [ [ "WordName", "x" ] ] ]
                  ]
                ]
              ]
            ]
          ]
        ]
      ]
    ]
  ],
  [ "LineBreak_Empty" ]
]